pkgs/development/compilers/solc/default.nix

```diff
@@ -1,28 +1,30 @@
 { stdenv, fetchzip, fetchgit, boost, cmake }:
 
+let rev = "194ff033ae44944ac59aa7bd3da89ba94ec5893c"; in
+let sha256 = "0gkg3nay0625qmhxxxax1d1c4dl554ri3pkwd12qfg6g1w6j04w7"; in
+let version = "0.4.12"; in
+
+let jsoncppURL = https://github.com/open-source-parsers/jsoncpp/archive/1.7.7.tar.gz; in
 let jsoncpp = fetchzip {
-  url = https://github.com/open-source-parsers/jsoncpp/archive/1.7.7.tar.gz;
+  url = jsoncppURL;
   sha256 = "0jz93zv17ir7lbxb3dv8ph2n916rajs8i96immwx9vb45pqid3n0";
 }; in
 
-let commit = "68ef5810593e7c8092ed41d5f474dd43141624eb"; in
-
-stdenv.mkDerivation rec {
-  version = "0.4.11";
+stdenv.mkDerivation {
+  inherit version;
```

Comment:
I believe this shouldn't be needed, I don't think `mkDerivation` cares about `version` attribute.